shared/src/commonMain/kotlin/org/hyperskill/app/stages/domain/model/Stage.kt

```diff
@@ -22,7 +18,7 @@ data class Stage(
     val projectStagesCount: Int,
 
     @SerialName("step")
-    val stepId: Long,
+    val stepId: Long?,
```

Comment:
Could you please add a link to the backend code, to verify types nullability.

Reply:
<img width="892" alt="Screenshot 2023-06-05 at 1 41 54 PM" src="https://github.com/hyperskill/mobile-app/assets/8586063/59eb83b2-f80f-4a96-ae0b-9234112da4f0">